torch/csrc/jit/passes/utils/alias_tracker.h

```diff
@@ -0,0 +1,119 @@
+#pragma once
+
+#include <torch/csrc/jit/ir.h>
+
+namespace torch {
+namespace jit {
+// class AliasTracker
+//
+// This class tracks the "A points to B" graph for all values, as well as
+// wildcards and writes. It is used by AliasDb to provide a higher-level API.
+class AliasTracker {
```

Comment:
Logically it's entirely internal, I separated them out physically because it's hard to test the graph implementation directly otherwise. 

I'm not that worried about random people using it—it's hard to construct without the actual analysis pass and otherwise there's no direct handle to it.

Reply:
Ok, sounds good! Will take a look at the next PR.